packages/drivers/routerlicious-driver/src/policies.ts

```diff
@@ -37,4 +55,11 @@ export interface IRouterliciousDriverPolicies {
      * Default: false
      */
     enableRestLess: boolean;
+    /**
+     * Configure if/how performance telemetry events are aggregated. Useful for reducing telemetry noise.
+     * Do not use if per-event data examination is needed.
+     * Aggregated will be logged as total sums along with a total count.
+     * Default: undefined
+     */
+    telemetryAggregation: ITelemetryAggregationPolicy | undefined;
```

Comment:
with the above comment, i'd just make this a boolean `disableTelemetryAggregation: true | undefined`, seems over complex as it is

Reply:
What would be your suggestions for thresholds, then? Assuming getVersions and getSnapshotTree would have similar thresholds, but readBlob would have a higher one? If we're going off ODSP's per-session events, then R11s has about 10x getSnapshotTree events per ODSP fetchSnapshot/ObtainSnapshot event. We could probably set the threshold for readBlob at 100?

Reply:
you are the best person to tune it. you understand the system, and can look code flow and data.  if you can, i'd call flush in a more scenario focused way, like maybe after getSnapshotTree, and then just set all the thresholds to big numbers

Reply:
in general consumers of this won't know, and won't care. maybe in the long term you get advance customer who care, but i'd rather not add complexity for the much larger pool of average and new customers